CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## v2.7.0 / 2022-11-25
+
+Note: Experimental VerticalPodAutoscaler metrics are considered deprecated in this release and will be removed in v2.9.0.
+Please use CustomResourceStateMetrics to gather metrics from VPA resources. See: #1718
+
+Note: Experimental CustomResourceState has been redesigned for better usability. See: #1847
```

Comment:
Since this is a breaking change note in the changelog, we should make it more explicit to the users. How about:

```md
Note: The naming convention of the experimental CustomResourceState has been reworked for better usability. The name of the CRD used to be interpolated into the name of the metric which made it impossible to aggregate a CRD across different versions. This was changed to have the GVK information represented as labels:

kube_myteam_io_v1_Foo_uptime -> kube_crd_uptime{group="myteam.io", kind="Foo", version="v1"} 
```

Reply:
Good point, I updated it!